src/rev_ai/models/__init__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
I like this, try to get it in all files. It helps on some weird Windows systems with funky encodings. I've had people not be able to install my modules because I forgot this.